swiftnav/src/coords/ecef.rs

```diff
@@ -0,0 +1,255 @@
+use nalgebra::{ArrayStorage, Vector3};
+use std::ops::{Add, AddAssign, Mul, MulAssign, Sub, SubAssign};
+
+use crate::coords::LLHDegrees;
+
+use super::{AzimuthElevation, Ellipsoid, LLHRadians, NED, WGS84};
+
+/// WGS84 Earth Centered, Earth Fixed (ECEF) Cartesian coordinates (X, Y, Z).
+///
+/// Internally stored as an array of 3 [`f64`] values: x, y, z all in meters
+#[derive(Copy, Clone, Debug, PartialEq, PartialOrd, Default)]
+pub struct ECEF(Vector3<f64>);
```

Comment:
It might be nice to impl a `From` on `Vector3` and also maybe a tuple like `(f64, f64, f64)`?